accounts-db/src/accounts_db.rs

```diff
@@ -4114,7 +4112,7 @@ impl AccountsDb {
             self.shrink_collect::<AliveAccounts<'_>>(store, &unique_accounts, &self.shrink_stats);
 
         // This shouldn't happen if alive_bytes/approx_stored_count are accurate
-        if Self::should_not_shrink(shrink_collect.aligned_total_bytes, shrink_collect.capacity) {
+        if Self::should_not_shrink(shrink_collect.alive_total_bytes as u64, shrink_collect.capacity) {
```

Comment:
I'm not really sure what this check is doing. It adds a page size to alive bytes and compares to the capacity of the original append vec. This data point does not appear to be present on mnb.

Reply:
This call to `should_not_shrink` is the same call we made prior to trying to shrink this append vec. This is a sanity check to make sure we still think we should've shrunk this one.